docs/requirements.txt

```diff
@@ -2,4 +2,5 @@ m2r2==0.2.7
 mistune==0.8.4
 sphinx==4.5.0
 toml==0.10.2
-sphinx-rtd-theme==1.0.0
\ No newline at end of file
+sphinx-rtd-theme==1.0.0
+pydantic==1.9.0
```

Comment:
why?

Reply:
I had initially removed poetry dependencies from the `.readthedocs.yml`, but I have now re-added them because that was the wrong choice.